### ISSUE TITLE
Continue development of EmulationStation.

### ISSUE THREAD

**Issue by aaronfranke** [Aloshi/EmulationStation#780]
There have only been 3 commits in the past 2 years. Please update EmulationStation! It doesn't install on Ubuntu 18.04 due to many conflicting packages.

A snap or AppImage package is also an option.

**Comment by probonopd** [comment]
Let me know if you'd like some assistance in making an AppImage version @Aloshi

**Comment by robertybob** [comment]
@aaronfranke use the Retropie version?

**Comment by aaronfranke** [comment]
I have that for my Pi but I'd also like to install ES on my x86 Ubuntu laptop.

**Comment by terzag** [comment]
> There have only been 3 commits in the past 2 years. Please update EmulationStation! It doesn't install on Ubuntu 18.04 due to many conflicting packages.
> 
> A snap or AppImage package is also an option.

The deb package available is quite old and it would be great to have support fir current distros but in the meantime, all the dependencies are still in the repos (they just changed version and name) and it's easily fixable.

Download the deb, extract it somewhere, go in the DEBIAN subdir and edit the control file. Replace the original dependencies list by that one (for Ubuntu 18.04):

`Depends: libasound2 (>= 1.0.16), libboost-filesystem1.65.0, libboost-locale1.65.0, libboost-system1.65.0, libc6 (>= 2.14), libcurl4 (>= 7.16.2), libfreeimage3, libfreetype6 (>= 2.2.1), libgcc1 (>= 1:4.1.1), libgl1-mesa-glx | libgl1, libsdl2-2.0-0 (>= 2.0.0), libstdc++6 (>= 4.6), fonts-droid-fallback`

On Ubuntu 18.10, same list but the libboost packages are at 1.67.0 instead of 1.65.0.

When it's done, delete the original .deb, open a terminal, cd to the dir where you've extracted it and rebuild the package (install the fakeroot and dpkg packages before if they're not):

fakeroot dpkg-deb -b emulationstation_amd64_latest

**Comment by aaronfranke** [comment]
> and it's easily fixable.

This is something that the EmulationStation devs should do then.

**Comment by probonopd** [comment]
Here is an [AppImage](https://appimage.org/) I have generated from the deb ingredients:

https://bintray.com/probono/AppImages/EmulationStation#files

Here is the recipe that produced it:

https://github.com/AppImage/AppImages/blob/c4fb1e02c17b92ae7db073e339e36cb2120bda23/recipes/EmulationStation.yml

I tested the resuting AppImage on Xubuntu 18.04.

**Comment by terzag** [comment]
> > and it's easily fixable.
> 
> This is something that the EmulationStation devs should do then.

Yes, but the project doesn't seem to be that active and maybe they don't have the resources to maintain deb packages anymore (it's available as a bonus anyway, it's usually the job of distros maintainers to provide specific packages). So, if you don't want to wait for an eventual updated deb, you can follow my instructions to build one that installs fine.

**Comment by probonopd** [comment]
If someone with commit rights in this repository is interested, we can make automatic continuous AppImage builds on [Travis CI ](https://travis-ci.org) right from the source - without Debian packaging. Would be much simpler than making debs and converting them to AppImage.

**Comment by robertybob** [comment]
@aaronfranke you can install RetroPie on Ubuntu ....

**Comment by terzag** [comment]
> @aaronfranke you can install RetroPie on Ubuntu ....

Is it available as an app you can start from the desktop or does it work only as a full/dedicated session?
